pennylane/qnode_new/__init__.py

```diff
@@ -15,16 +15,9 @@
 This package contains the new-style QNodes.
 
 .. currentmodule:: pennylane.qnode_new
-
-.. autosummary::
-    :toctree: api
-
-    qnode
-    jacobian
-    qubit_parameter_shift
-    cv_parameter_shift
 """
-from .qnode import QNode, QuantumFunctionError
+from .cv import CVQNode
+from .decorator import qnode, QNode
```

Comment:
The previously mentioned (comment for ``decorator.py``) ambiguity of ``QNode`` being a function and not a class can be spotted here. If only looking at this list of imports, which ``QNode`` is the class actually? With the function renamed, I believe the class could still be called ``QNode`` (but renaming it to ``BaseQNode`` sounds alright as well, it might just confuse some people about the change)

Reply:
See https://github.com/XanaduAI/pennylane/pull/420#discussion_r346085550